### ISSUE TITLE
Email does not arrive on restored backups

### ISSUE THREAD

**Issue by SimplyCorbett** [mailcow/mailcow-dockerized#3822]
I have had to backup and restore mailcow on four separate occasions over the last year and a half. Mainly because I was moving servers.

In all cases from all indications the restore went smoothly, however email never arrived on any of the restored servers.

This is using the backup/restore script provided by mailcow.

A full reinstall was required in order to receive email on all occasions.

I have around 100 aliases on my personal account, so not sure if that has anything to do with it. Anyway, this is more of a FYI thing. I didn't see any bugs in the logs, so not sure how useful this information will be. And I've already started fresh again so no point in restoring the backups now.

**Comment by andryyy** [comment]
You did something wrong then.

But what are we supposed to do without any information? :/

**Comment by SimplyCorbett** [comment]
I just tried this again as I'm moving servers. This time it wouldn't show any email accounts at all with a full restore. Also tried using the same mailcow.conf eventually. Ended up just copying the entire root directory over and regenerating grub.

Tried this with several reinstalls on the new server to rule out changes on my end.

I'm not sure if this is something I'm doing wrong or if there is a bug in the backup/restore functionality. I'd imagine the restore functionality is only used in a blue moon so isn't mass tested.

Anyway, as I've fully setup the new server now with the manual migration there isn't any reason to re-open this ticket. I'd love to see further documentation in the backup/restore section about restoring on a fresh installation though.
